packages/nodes-base/nodes/MongoDb/MongoDb.node.json

```diff
@@ -1,6 +1,6 @@
 {
 	"node": "n8n-nodes-base.mongoDb",
-	"nodeVersion": "1.0",
+	"nodeVersion": "1.1",
```

Comment:
This won't be needed as we take the info from the node

Reply:
Removed.